### DISCUSSION TITLE
How to set the supply region and related prices for the APP

### DISCUSSION THREAD

**Discussion by tian13019692331** [fastlane/fastlane#21985] [Q&A]
How to set the supply region and related prices for the APP

**Comment by jennie258fitz** [comment]
Setting the supply region and related prices for your app involves a few key steps, depending on the platform you’re using. Here’s a general guide for both the Apple App Store and Google Play Store:

Apple App Store
Access App Store Connect:
Log in to App Store Connect.
Select Your App:
From the Apps section, select your app.
Navigate to Pricing and Availability:
In the sidebar under Monetization, click Pricing and Availability.
Set Base Country or Region:
In the Price Schedule section, click Add Pricing.
Select a base country or region. Apple uses this base to provide comparable prices across other storefronts.
Choose Price Points:
Select a price for your base country or region. Apple will automatically generate prices for other regions based on [NY State of Health](https://www.ny-stateofhealth.com/) this selection.
You can manually adjust prices for specific regions if needed.
Google Play Store
Access Google Play Console:
Log in to Google Play Console.
Select Your App:
From the All Apps section, select your app.
Navigate to Pricing & Distribution:
Go to the Pricing & Distribution section.
Set Pricing:
Set the price for your app. You can choose to apply the same price globally or set different prices for different regions.
Select Distribution Regions:
Choose the countries or regions where you want your app to be available. You can also exclude specific regions if needed.
Tips for Both Platforms
Monitor Exchange Rates and Taxes: Both platforms adjust prices based on exchange rates and local taxes. Regularly review and update your prices to ensure they remain competitive.
Use Automated Tools: Consider using automated tools or APIs provided by the platforms to manage pricing and availability efficiently.
Stay Informed: Keep up with platform updates and changes in pricing policies to ensure compliance and optimize your app’s revenue.
